MFaaP.MFWSClient/MFWSVaultValueListItemOperations.cs

```diff
@@ -69,5 +70,47 @@ public Results<ValueListItem> GetValueListItems(int valueListId, string nameFilt
 				.GetAwaiter()
 				.GetResult();
 		}
+
+
+		/// <summary>
+		/// Creates a new item with the <see cref="newItemName"/> in the value list with id <see cref="valueListId"/>.
+		/// </summary>
+		/// <param name="valueListId">The Id of the value list to create the new item in.</param>
+		/// <param name="newItemName">Name of the new value list item to create</param>
+		/// <param name="token">A cancellation token for the request.</param>
+		/// <returns>The newly created value list item</returns>
+		public async Task<ValueListItem> CreateValueListItemAsync(int valueListId, string newItemName, CancellationToken token = default(CancellationToken))
+        {
+			var newValueListItem = new ValueListItem
+            {
+				Name		= newItemName,
+				ValueListID = valueListId
+            };
+
+			// Create the request.
+			var request = new RestRequest($"/REST/valuelists/{valueListId}/items");
+			request.AddJsonBody(newValueListItem);
+
+			// Make the request and get the response.
+			var response = await this.MFWSClient.Post<ValueListItem>(request, token)
+				.ConfigureAwait(false);
+
+			return response.Data;
+        }
+
+		/// Creates a new item with the <see cref="newItemName"/> in the value list with id <see cref="valueListId"/>.
+		/// </summary>
+		/// <param name="valueListId">The Id of the value list to create the new item in.</param>
+		/// <param name="newItemName">Name of the new value list item to create</param>
+		/// <param name="token">A cancellation token for the request.</param>
+		/// <returns>The newly created value list item</returns>
+		public ValueListItem CreateValueListItem(int valueListId, string newItemName, CancellationToken token = default(CancellationToken))
```

Comment:
Please update the method to be named "AddValueListItem", akin to the [COM API](https://www.m-files.com/api/documentation/latest/index.html#MFilesAPI~VaultValueListItemOperations~AddValueListItem.html) method.